tests/e2e/specs/pageProductOverview_spec.js

```diff
@@ -59,7 +59,7 @@ describe('Product overview page', () => {
   it('Displays a message when the product list is empty', () => {
     cy.visit('/products/accessories');
     cy.get('span[data-test=sort-selector]')
-      .should('not.exist');
+      .should('not.exist', { timeout: Cypress.config('graphqlTimeout') });
```

Comment:
It will not exist from the beginning, so this timeout in unnecessary. Actually, the assertion itself is doubtful. I'll move it to the end, after the `No results found.` assertion.